nuget/nunit-project-loader.nuspec

```diff
@@ -17,8 +17,4 @@
     <tags>nunit test testing tdd runner</tags>
     <copyright>Copyright (c) 2016 Charlie Poole</copyright>
   </metadata>
-  <files>
-    <file src="../../LICENSE.txt" />
-    <file src="nunit-project-loader.dll" target="tools"/>
-  </files>
```

Comment:
Removing files and leaving only metadata in the nuspec puts both chocolatey and nuget on the same footing. Files to be included are supplied by the script.